recipes/imapclient/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{%set name = "IMAPClient" %}
+{%set version = "1.0.2" %}
+{%set compress_type = "tar.gz" %}
```

Comment:
You as the maintainer have the final say. I am just expressing my opinion. (And if my script becomes an actual tool for conda-forge you will need to choose a style to be able to use it :wink:)